observability/logging/logging-6.2/log6x-clf-6.2.adoc

```diff
@@ -0,0 +1,122 @@
+
+Inputs:: Select log messages to be forwarded. Built-in input types `application`, `infrastructure` and `audit` forward logs from different parts of the cluster. You can also define custom inputs.
+
+Outputs:: Define destinations to forward logs to. Each output has a unique name and type-specific configuration.
+
+Pipelines:: Define the path logs take from inputs, through filters, to outputs. Pipelines have a unique name and consist of a list of input, output and filter names.
+
+Filters:: Transform or drop log messages in the pipeline. Users can define filters that match certain log fields and drop or modify the messages. Filters are applied in the order specified in the pipeline.
+
+[id="clf-inputs_6-2_{context}"]
+=== Inputs
+
+Inputs are configured in an array under `spec.inputs`. There are three built-in input types:
+
+application:: Selects logs from all application containers, excluding those in infrastructure namespaces.
+
+infrastructure:: Selects logs from nodes and from infrastructure components running in the following namespaces:
+** `default`
+** `kube`
+** `openshift`   
+** Containing the `kube-` or `openshift-` prefix
+
+audit:: Selects logs from the OpenShift API server audit logs, Kubernetes API server audit logs, ovn audit logs, and node audit logs from auditd.
+
+Users can define custom inputs of type `application` that select logs from specific namespaces or using pod labels.
+
+[id="clf-outputs_6-2_{context}"]
+=== Outputs
+
+Outputs are configured in an array under `spec.outputs`. Each output must have a unique name and a type. Supported types are:
+
+azureMonitor:: Forwards logs to Azure Monitor.
+cloudwatch:: Forwards logs to AWS CloudWatch.
+//elasticsearch:: Forwards logs to an external Elasticsearch instance.
+googleCloudLogging:: Forwards logs to Google Cloud Logging.
+http:: Forwards logs to a generic HTTP endpoint.
+kafka:: Forwards logs to a Kafka broker.
+loki:: Forwards logs to a Loki logging backend.
+lokistack:: Forwards logs to the logging supported combination of Loki and web proxy with {Product-Title} authentication integration. LokiStack's proxy uses {Product-Title} authentication to enforce multi-tenancy
+otlp:: Forwards logs using the OpenTelemetry Protocol.
+splunk:: Forwards logs to Splunk.
+syslog:: Forwards logs to an external syslog server.
+
+Each output type has its own configuration fields.
+
+include::modules/log6x-configuring-otlp-output.adoc[leveloffset=+1]
+
+[id="clf-pipelines_6-2_{context}"]
+=== Pipelines
+
+Pipelines are configured in an array under `spec.pipelines`. Each pipeline must have a unique name and consists of:
+
+inputRefs:: Names of inputs whose logs should be forwarded to this pipeline.
+outputRefs:: Names of outputs to send logs to.
+filterRefs:: (optional) Names of filters to apply.
+
+The order of filterRefs matters, as they are applied sequentially. Earlier filters can drop messages that will not be processed by later filters.
+
+[id="clf-filters_6-2_{context}"]
```

Comment:
The 6.0 preview does not include Configuring OTLP output, but this is OK?

Reply:
Yes, this is consistent with https://docs.openshift.com/container-platform/4.17/observability/logging/logging-6.0/log6x-loki.html